linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_command/rule.yml

```diff
@@ -23,6 +23,11 @@ severity: medium
 identifiers:
     cce@rhel8: 80940-0
 
+references:
+    disa: '56'
+    nist: AC-11(b)
```

Comment:
This appears to be aligned with AC-11(a), not AC-11(b). Realize this was likely generated by a script absorbing whatever DISA had... but we don't want to inherit inaccurate mappings.

AC-11(a) -- enable lock
AC-11(b) -- retain until user reauthenticates
https://nvd.nist.gov/800-53/Rev4/control/AC-11

Thinking through this, it appears this control might meet AC-11(a) and (b). What do you think?

Reply:
@shawndwells maybe? I guess it does the last part of AC-11(a).... for a timeout and autolock, we need a new rule to completely resolve this.

Reply:
@shawndwells fixed